File: 125-valid-palindrome/valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string alpha = "";
        for (auto ele: s){
            if(isalnum(ele)){
                alpha += tolower(ele);
            }
        }
        cout<<alpha<<endl;
        string rev = alpha;
        reverse(rev.begin(), rev.end());
        if(alpha == rev) return true;
        return false;
    }
};

File: 125-valid-palindrome/valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        i = 0
        j = len(s)-1
        while(i <= j):
            if not (s[i].isdigit() or s[i].isalpha()):
                i += 1
            elif not (s[j].isdigit() or s[j].isalpha()):
                j -= 1
            elif s[i].lower() != s[j].lower():
                return False
            else:
                i += 1
                j -= 1

        return True
                
        